openassessment/xblock/static/js/src/lms/oa_base.js

```diff
@@ -66,11 +72,33 @@ OpenAssessment.BaseView.prototype = {
      * @param {element} parentElement JQuery selector for the container element.
      */
     setUpCollapseExpand: function(parentElement) {
-        parentElement.on('click', '.ui-toggle-visibility__control', function(eventData) {
-                var sel = $(eventData.target).closest('.ui-toggle-visibility');
-                sel.toggleClass('is--collapsed');
-            }
-        );
+        var view = this;
+
+        $('.' + view.SLIDABLE_CONTROLS_CLASS, parentElement).each(function() {
+            $(this).on("click", function(event) {
+                event.preventDefault();
+
+                var $slidableControl = $(event.target).closest('.' + view.SLIDABLE_CONTROLS_CLASS);
+
+                var $container = $slidableControl.closest('li');
+                var $toggleButton = $slidableControl.find('.' + view.SLIDABLE_CLASS);
+                var $panel = $slidableControl.next('.' + view.SLIDABLE_CONTENT_CLASS);
+
+                if ($toggleButton.hasClass('is--showing')) {
+                    $panel.slideUp();
+                    $toggleButton.attr('aria-expanded', 'false');
+                    $toggleButton.removeClass('is--showing');
```

Comment:
Yep, as we talked about, I'm going to clean it up and just put it in the encompassing container.